azurerm/resource_arm_relay_namespace_test.go

```diff
@@ -28,16 +29,66 @@ func TestAccAzureRMRelayNamespace_basic(t *testing.T) {
 					resource.TestCheckResourceAttrSet(resourceName, "secondary_connection_string"),
 					resource.TestCheckResourceAttrSet(resourceName, "primary_key"),
 					resource.TestCheckResourceAttrSet(resourceName, "secondary_key"),
+					resource.TestCheckResourceAttr(resourceName, "sku_name", "Standard"),
 				),
 			},
 			{
 				ResourceName:      resourceName,
 				ImportState:       true,
-				ImportStateVerify: true,
+				ImportStateVerify: false,
```

Comment:
Why is has this been changed?

Reply:
Has to be false because the state will never match with the new attribute...

Reply:
as above. if they are computed and you are setting both on read they will.